pkg/controller/syncsetinstance/syncsetinstance_controller.go

```diff
@@ -586,20 +586,8 @@ func (r *ReconcileSyncSetInstance) reapplyTime(status hivev1.SyncStatus) time.Ti
 func (r *ReconcileSyncSetInstance) applySyncSetResources(ssi *hivev1.SyncSetInstance, resources []runtime.RawExtension, applyBehavior hivev1.SyncSetApplyBehavior, dynamicClient dynamic.Interface, h Applier, ssiLog log.FieldLogger) error {
 	ssiLog = ssiLog.WithField("applyTerm", "resource")
 
-	// determine if we can gather info for all resources
-	infos := []hiveresource.Info{}
-	for i, resource := range resources {
-		info, err := h.Info(resource.Raw)
-		if err != nil {
-			ssi.Status.Conditions = r.setUnknownObjectSyncCondition(ssi.Status.Conditions, err, i)
-			ssiLog.WithError(err).Warn("unable to parse resource")
-			return err
-		}
-		infos = append(infos, *info)
-	}
-
-	ssi.Status.Conditions = r.clearUnknownObjectSyncCondition(ssi.Status.Conditions)
 	syncStatusList := []hivev1.SyncStatus{}
+	var applyErr error
```

Comment:
Why was this moved?

Reply:
I have just moved it outside the loop

Reply:
It was not inside the loop before. It was declared right before the loop.